phpBB/adm/style/admin.css

```diff
@@ -1653,6 +1653,107 @@ fieldset.submit-buttons legend {
 	}
 }
 
+/* Mentions and mention dropdown
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+.atwho-view { /* mention-container */
+	text-align: left;
+	background-color: #ffffff;
+	border-radius: 2px;
+	box-shadow:
+		0 3px 1px -2px rgba(0, 0, 0, 0.2),
+		0 2px 2px 0 rgba(0, 0, 0, 0.14),
+		0 1px 5px 0 rgba(0, 0, 0, 0.12);
+	position: absolute;
+	z-index: 999;
+	top: 20px;
```

Comment:
Sorry my bad this may not be necessary as I just added it to give the page some spacing, we can refine it in RC